src/user/serializers.py

```diff
@@ -1129,7 +1130,16 @@ def get_open_access_pct(self, author):
             / total_paper_count
         )
 
-    def get_reputations(self, author):
+    def get_reputation(self, author):
+        return {
+            "hub_id": 1,
+            "hub_name": "Cell Biology",
+            "hub_slug": "slug-1",
+            "score": 1982000,
```

Comment:
This value is greater than the top bin value?

Reply:
Will update the bins in the next PR.

Reply:
Nvm, updated.